web/src/repo/blob/Blob.tsx

```diff
@@ -197,8 +200,27 @@ export class Blob extends React.Component<BlobProps, BlobState> {
                 dom: domFunctions,
             })
         )
+        const j2d = (ev: MouseEvent) => {
```

Comment:
For readability:

```suggestion
        const jumpToDefinition = (ev: MouseEvent) => {
```

What's more, our codebase currently uses `goToDefinition`, not `jumpToDefinition`, any reason to break  consistency here (and go with `singleClickGoToDefinition`, `goToDefinition`, `goToDefinitionAction`)?

Reply:
:+1: Renamed to `goToDefinition` et al.